firebase-sessions/test-app/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
If we are planning to not have proguarding enabled, do we even need to have this file?

Reply:
Removed.